common/app/common/NewNavigation.scala

```diff
@@ -13,6 +13,7 @@ object NewNavigation {
 
   val PrimaryLinks = List(headlines, opinion, sport, culture, lifestyle)
   val topLevelSections = List(News, Opinion, Sport, Arts, Life)
+  val brandExtensions = List(jobs, dating, holidays, ukMasterClasses)
```

Comment:
Could you update [this](https://github.com/guardian/frontend/blob/master/common/app/common/NewNavigation.scala#L229) to use the brand extensions list a long with the other list (you can concatenate it?